formatter.go

```diff
@@ -425,6 +432,14 @@ func (f *DefaultFormatter) formatValue(value interface{}) string {
 	return sq.Sdump(value)
 }
 
+func extractFloat(value interface{}) float64 {
+	if v, ok := value.(float32); ok {
+		return float64(v)
+	}
+
+	return value.(float64)
+}
```

Comment:
* Please move it below, near other `extract*` funcs.
* Other extract funcs have default branch, let's add it here too.
* Other extract funcs are switches, could we make this one too?

Reply:
ok, will do the change in this PR

Reply:
updated https://github.com/gavv/httpexpect/pull/220/commits/d418ccf01e2e52ddd1893ecf57fef1fe888badac